docs/firmware/device-sdk/authentication/guide-auth.md

```diff
@@ -2,8 +2,7 @@
 title: Guide
 sidebar_position: 2
 ---
-import ComingSoon from '/docs/partials/coming-soon-centered.md'
+Work through the [Getting Started Guide](https://docs.golioth.io/getting-started) to create device credentials and see where to add those credentials to your Golioth application.  After credentials are entered into the configuration file, there is no further direct interaction with the credentials.  Credentials will be handled by the Golioth System Client in the background.

```

Comment:
I think we could be more specific and point to https://docs.golioth.io/getting-started/authorize-devices instead of just the getting-started (assuming that users who wanted to get started already got started, and now they're digging into more details)